solc/CommandLineInterface.cpp

```diff
@@ -329,7 +329,7 @@ void CommandLineInterface::handleEWasm(string const& _contractName)
 		{
 			sout() << "EWasm text: " << endl;
 			sout() << m_compiler->eWasm(_contractName) << endl;
-			sout() << "EWasm binary (hex): " << m_compiler->eWasmObject(_contractName).toHex() << endl;
+			sout() << "EWasm binary (hex): " << std::uppercase << m_compiler->eWasmObject(_contractName).toHex() << endl;
```

Comment:
`toHex()` returns a string and uses `dev::toHex` internally, which should use an explicit case. Did you observe this somewhere?

Reply:
Sorry, somehow didn't notice `toHex()`, which indeed does this correctly. I had a follow up PR which changes the other `uppercase` to `nouppercase` to match all the other cases of `toHex`.